### ISSUE TITLE
gestione codice di uscita spid-sp-test e messaggio all'utente

### ISSUE THREAD

**Issue by peppelinux** [italia/spid-saml-check#182]
È necessario valutare lo status code di uscita di spid-saml-check.
Quando diverso da 0 sarebbe bene rappresentare a video (dialog errore) stdout e stderr del processo

**Comment by peppelinux** [comment]
Di fatto già spid-saml-check stampa stdout e stderr di spid-sp-test, basta fare un attach sul container.
Sarebbe comunque utile catturarli ed esporli nella UI all'utente se tu fossi daccordo

![image](https://user-images.githubusercontent.com/1297620/135114818-926aeef0-4365-4403-aca2-4eae1f82de4c.png)

**Comment by damikael** [comment]
Certo, come già d'accordo, la feature è da rilasciare con i prossimi aggiornamenti.
